README.md

```diff
@@ -0,0 +1,9 @@
+# Source code for Gradle manupilation plugins
+
+## Alignment
```

Comment:
Given we might be doing more than alignment here I think we need to keep this more generic e.g.
```suggestion
## Analysis
```

Reply:
First plugin renamed as `analyzer`.